### DISCUSSION TITLE
Graphite Query Editor Dropdown- Improve scrolling experience

### DISCUSSION THREAD

**Discussion by bob454522** [grafana/grafana#41017] [Data Sources]
**What happened**:
When using the (auto-populated) dropdown menu to select parts of your graphite query,  if your query is towards the bottom of the frame, some options can not be scrolled to / not-visible.

**What you expected to happen**:  Additional scroll "blank space" provides so that the frame can be scrolled down further, thus allowing the entire (auto-populated) dropdown menu to be visible.

**How to reproduce it (as minimally and precisely as possible)**:
best to show a screen cap:
 (note there are several options below the Vars in this drop down menu that i cant get to, also the browser window is 1306px width , by 1016px tall  , so this is not a small browser window.  Browser window size does not affect this though (occurs at all browser window sizes):

![dropDown](https://user-images.githubusercontent.com/60830628/139123688-bdd3ac95-a50b-4331-8ebe-258a2b3d245c.gif)

**Anything else we need to know?**:
(would be nice if we could disable auto-population of user defined variables,  or place them (sort) at the BOTTOM of the auto-populated entries.  (ie all the $var1 , $var2, $var3  - have them at the bottom of the drop-down, as the auto-populated values are more frequently used/clicked,  on average)

**Environment**:
- Grafana version:  8.2.2
- Data source type & version:  Graphite (v0.9.14)
- OS Grafana is installed on: Debian 8
- User OS & Browser: Win10 on Chrome
- Grafana plugins: N/A
- Others:
thank you- LOVE GRAFANA!

**Comment by ifrost** [comment]
This should be addressed by https://github.com/grafana/grafana/pull/42446

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
